scout/server/blueprints/cases/controllers.py

```diff
@@ -587,21 +573,48 @@ def _get_decorated_var(var_obj: dict) -> dict:
                 continue
             if case_key == "partial_causatives":
                 var_obj["phenotypes"] = case_obj["partial_causatives"][var_id]
-            evaluated_variants_by_type[eval_category].append(_get_decorated_var(var_obj=var_obj))
+            evaluated_variants_by_type[eval_category].append(
+                _get_decorated_var(var_obj=var_obj, institute_obj=institute_obj, case_obj=case_obj)
+            )
 
-    # Collect all evaluated variants except causative, partial causative and suspected variants
     for var_obj in store.evaluated_variants(
         case_id=case_obj["_id"], institute_id=institute_obj["_id"]
     ):
-        for eval_category, variant_key in CASE_REPORT_VARIANT_TYPES.items():
-            if var_obj.get(variant_key):
-                evaluated_variants_by_type[eval_category].append(
-                    _get_decorated_var(var_obj=var_obj)
-                )
+        _partition_var_by_type(evaluated_variants_by_type, var_obj, institute_obj, case_obj)
 
     data["variants"] = evaluated_variants_by_type
 
 
+def _get_decorated_var(var_obj: dict, institute_obj: dict, case_obj: dict) -> dict:
+    """Decorate a variant object for display using the variant controller"""
+    return variant_decorator(
+        store=store,
+        variant_id=None,
+        institute_id=institute_obj["_id"],
+        case_name=case_obj["display_name"],
+        variant_obj=var_obj,
+        add_other=False,
+        get_overlapping=False,
+        variant_type=var_obj["category"],
+        institute_obj=institute_obj,
+        case_obj=case_obj,
+    )["variant"]
+
+
+def _partition_var_by_type(
```

Comment:
```suggestion
def _set_evaluated_variants_by_type(
```

Feels more self-explanatory to me

Reply:
Hm, only one variant there so the suggested name would be a little misleading unless we wrap the whole loop in the function. Which might be a tad quicker, but these shouldn't be many. Set is generic, for good and bad, but point taken. I'll elaborate..